pkg/restic/restorer.go

```diff
@@ -84,14 +91,16 @@ func newRestorer(
 	return r
 }
 
-func (r *restorer) RestorePodVolumes(restore *velerov1api.Restore, pod *corev1api.Pod, sourceNamespace, backupLocation string, log logrus.FieldLogger) []error {
-	// get volumes to restore from pod's annotations
-	volumesToRestore := GetPodSnapshotAnnotations(pod)
+func (r *restorer) RestorePodVolumes(data RestoreData) []error {
+	volumesToRestore := GetVolumesForPod(data.PodVolumeBackups, data.Pod)
 	if len(volumesToRestore) == 0 {
```

Comment:
let's add a comment explaining the backwards-compat logic here, plus a `// TODO(2.0)` to remove the annotations code.